composer.json

```diff
@@ -19,20 +19,19 @@
   ],
   "require": {
     "php": "^7.0",
-    "composer/installers": "^1.0.20",
-    "drupal-composer/drupal-scaffold": "^2.0.1",
-    "cweagans/composer-patches": "~1.0",
-    "drush/drush": "~8.0",
-    "drupal/console": "~1.0",
+    "composer/installers": "^1.2",
+    "drupal-composer/drupal-scaffold": "^2.2",
+    "cweagans/composer-patches": "^1.6",
+    "drush/drush": "~8.1.16",
+    "drupal/console": "^1.0.2",
     "drupal/admin_toolbar": "~1.0",
     "drupal/config_devel": "~1.0-rc1",
     "drupal/config_installer": "~1",
     "drupal/entity_reference_revisions": "1.x-dev",
     "drupal/field_collection": "~1.0-alpha1",
     "drupal/inline_entity_form": "~1.0-beta1",
     "drupal/jsonb": "~1.0-beta1",
-    "drupal/paragraphs": "1.x-dev",
-    "drupal-composer/drupal-project": "8.x-dev",
```

Comment:
This project is not really setup to be a dependency and is more of a skeleton from drupal projects. I have moved the small amount of code that we were drawing on into `scripts/composer/ScriptHandler.php`.